packages/api/src/makeMergedSchema/makeMergedSchema.ts

```diff
@@ -155,7 +155,7 @@ export const makeMergedSchema = ({
   services: Services
   schemaDirectives?: IExecutableSchemaDefinition['schemaDirectives']
 }) => {
-  const typeDefs = mergeTypes(
+  const typeDefs = mergeTypeDefs(
     [rootSchema.schema, ...Object.values(schemas).map(({ schema }) => schema)],
```

Comment:
@peterp, We get `No overload matches this call.` from this line.
Wrapping it up in lodash.merge fixes the error for this line.